Documentation/PackedAssetBuilds.md

```diff
@@ -18,47 +46,94 @@ Inside the installer select the version to use.  Install version **2021.1.11.793
 
 > REBOOT (or logout / login). Otherwise, the environment settings won't get picked up for any builds in Visual Studio. They will only apply to command-line builds, and only for any command-line windows that have been opened after the installer finishes.
 
-### Build profile build and process assets
+### Build non-monolithic profile builds of the Engine
 
-Build and run MPS as per the [README.md](../README.md) and ensure all assets are built.
+Build a regular profile build of the game as per the [README.md](../README.md) in an engine-centric way. Be sure to build from the source engine and not the installer. Here is an example:
 
-### Test the profile build
+Clone the engine from source.
 
-* Open the game in editor
-    * load `NewStarBase` level
-    * Verify that game can launch and connect to local server from editor
-* Validate local game launcher can connect to local server
+```shell
+C:\git> git clone https://github.com/o3de/o3de
+```
+
+Navigate to `C:\git\o3de`.
+Create a build folder for non-monolithic build of the engine, for example `C:\git\o3de\build_non_mono`.
+
+```shell
+C:\git\o3de> mkdir build_non_mono
+```
+> The location and the name of the build folder is optional.
+
+Configure the engine in a non-monolithic mode with o3de-multiplayersample project.
+
+```shell
+C:\git\o3de\build_non_mono> cmake .. -DLY_MONOLITHIC_GAME=OFF -DLY_PROJECTS="C:\git\o3de-multiplayersample"
+```
+
+> `-DLY_MONOLITHIC_GAME=OFF` is the default value but for clarity it's specified here explicitly.
+
+Build the Editor. This will compile the project and necessary gems to produce the required game assets.
+
+```shell
+C:\git\o3de\build_non_mono> cmake --build . --target Editor --config profile
+```
+
+Run the Asset Processor with o3de-multiplayersample and let all the assets get processed.
+
+```shell
+C:\git\o3de\build_non_mono> .\bin\profile\AssetProcessor.exe --project-path C:\git\o3de-multiplayersample
+```
 
 ### Build AssetBuilder
 
-You will need to build the [AssetBundler](https://www.o3de.org/docs/user-guide/packaging/asset-bundler/overview/) tool if not built.
+You will need to build the [AssetBundler](https://www.o3de.org/docs/user-guide/packaging/asset-bundler/overview/) tool if not built to create game bundles (.pak files).
 
 For example:
 ```shell
-cmake --build build\windows --target AssetBundler --config profile -- /m /nologo
+C:\git\o3de\build_non_mono> cmake --build . --target AssetBundler --config profile
 ```
 
+### Test the profile build
+
+* Open the game in editor
+    * load `NewStarBase` level
+    * Verify that game can launch and connect to local server from editor
+* Validate local game launcher can connect to local server
+
+
```

Comment:
I recommend putting the ideal steps at the top and move the testing to a `Troubleshooting` section at the bottom.  If all a user wants to do is package MPS they should just be able to follow the list at the top, but if something goes wrong they can look further down the page

Reply:
I just removed this section. The main readme already covers this.